index.d.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { ViewProperties } from 'react-native';
+
+export interface BlurViewProperties {
+  blurType: 'xlight' | 'light' | 'dark'
+    // tvOS only
+    | 'extraDark' | 'regular' | 'prominent';
+  blurAmount?: number; // 0 - 100
```

Comment:
Hi @Kureev, the only possible way to do it currently would be to have a enum like:
```
blurAmount?: 0 | 1 | 2 | 3 | ... | 99 | 100;
```
This issue on Typescript is exactly what we are looking for, but it looks like there is currently no clean way to do it, appart using an enum https://github.com/Microsoft/TypeScript/issues/15480 👀